Resources/Prototypes/_LostParadise/Loadouts/Jobs/security.yml

```
@@ -0,0 +1,138 @@
+#Снаряжение
```

yamllint

[warning] missing starting space in comment

(comments)